doc/api/cli.md

```diff
@@ -243,6 +243,15 @@ Load an OpenSSL configuration file on startup. Among other uses, this can be
 used to enable FIPS-compliant crypto if Node.js is built with
 `./configure --openssl-fips`.
 
+### `--use-openssl-ca`, `--use-bundled-ca`
+<!-- YAML
+added: v7.3.0
+-->
+
+Use OpenSSL's default CA store or use bundled Mozilla CA store as supplied by
+current NodeJS version. Using OpenSSL store allows external modifications
+while bundled store is fixed at compile time. See `SSL_CERT_DIR` and
```

Comment:
"while bundled store" -> ", the bundled store" -- as-is, the text can be misunderstood to mean that the ossl store can be modified only if the bundled store is fixed. I suggest adding some text to the effect that the ossl store exists on most Linux distributions but may not exist elsewhere, and the certificates in it are maintained and decided by the distribution packagers unless explicitly reconfigured by the system admin, and also that the bundled store is identical for all Node.js binaries across all platforms.